### PR TITLE
Tao 1763 response mapping serialization

### PR DESCRIPTION
**Pull request by ssipasseuth**

### CODE REVIEW

**Comment by ssipasseuth** [comment]
@bugalot , if you are too busy, please just take a look at the standard compliance aspect of the new responseDeclaration template : https://github.com/oat-sa/extension-tao-itemqti/pull/479/files#diff-deb5ca4b606a5457b320c7eb24d8c655
Then you can forward the code review it to @jsconan.
Thanks !

**Comment by krampstudio** [comment]
The test doesn't run in PhantomJs using 

`grunt connect qunit:single --test /taoQtiItem/views/js/test/qtiXmlRenderer/test.html`

File: views/js/test/qtiXmlRenderer/test.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Qti XML Renderer Test</title>
+        <base href="../../../../../../tao/views/" />
+        <link rel="stylesheet" type="text/css" href="js/lib/qunit/qunit.css">
+        <script type="text/javascript" src="js/lib/require.js"></script>
+        <script type="text/javascript" src="js/lib/qunit/qunit.js"></script>
+        <script type="text/javascript" src="js/lib/qunit/qunit-parameterize.js"></script>
+        <script type="text/javascript" src="js/lib/blanket/blanket.min.js"  data-cover-flag="branchTracking"  data-cover-only="ui/component.js"></script>
```
Comment (krampstudio) [inline views/js/test/qtiXmlRenderer/test.html]: the coverage filter is wrong

**Comment by ssipasseuth** [comment]
I fixed the coverage and added more samples to improve the coverage. Ready for a new review.

**Pull request merged**